### PR TITLE
set vanity url code

### PR DESCRIPTION
**Pull request by muratvastark**
I wanted to change my server's special invitation, but it was not available, I wanted to add :3

### CODE REVIEW

**Comment by ImRodry** [comment]
what

File: typings/index.d.ts [hunk 1]
```diff
@@ -188,6 +188,7 @@ export abstract class AnonymousGuild extends BaseGuild {
   public vanityURLCode: string | null;
   public verificationLevel: VerificationLevel;
   public bannerURL(options?: StaticImageURLOptions): string | null;
+  public setVanityURL(code: string, reason?: string): string | null;
```
Comment (imranbarbhuiya) [inline typings/index.d.ts:191]:
```suggestion
  public setVanityURL(code: string, reason?: string): Promise<Guild>;
```

**Review by imranbarbhuiya — CHANGES_REQUESTED** [review]

**Comment by kyranet** [comment]
It seems like you made this PR from the wrong base, like if you wanted to PR to v13 instead of main, there are many unrelated changes here.

**Comment by muratvastark** [comment]
> Lol u closed

https://github.com/discordjs/discord.js/pull/7456

I moved it to v13 branch

**Pull request closed without merging**